### PR TITLE
Update all ru_RU locale and fix some typos in en_US

### PR DESCRIPTION
**Pull request by ppv1979**

### CODE REVIEW

File: locale/en_US/locale.xml [hunk 1]
```diff
@@ -349,7 +349,7 @@ Note that the OAI interface can be disabled for this site by editing the OJS sys
 </ul>]]></message>
 
 	<!-- System Information -->
-	<message key="admin.overwriteConfigFileInstructions"><![CDATA[<h4>NOTE!</div>
+	<message key="admin.overwriteConfigFileInstructions"><![CDATA[<h4>NOTE!</h4>
```
Comment (mtub) [inline locale/en_US/locale.xml]: I thought I had that one fixed some time ago… don't know why it's still in here, but good catch!

File: locale/en_US/locale.xml [hunk 2]
```diff
@@ -394,7 +394,7 @@ Authors who publish with this journal agree to the following terms:
 	<li>Authors are permitted and encouraged to post their work online (e.g., in institutional repositories or on their website) prior to and during the submission process, as it can lead to productive exchanges, as well as earlier and greater citation of published work (See <a target="_new" href="http://opcit.eprints.org/oacitation-biblio.html">The Effect of Open Access</a>).</li>
 </ol>
 
-<h4>Proposed Policy for Journals That Offer Delayed Open Access</h4>
+<h4>2. Proposed Policy for Journals That Offer Delayed Open Access</h4>
```
Comment (mtub) [inline locale/en_US/locale.xml]: :+1:

File: locale/ru_RU/admin.xml [hunk 3]
```diff
@@ -7,14 +7,13 @@
   * Copyright (c) 2013-2016 Simon Fraser University Library
   * Copyright (c) 2003-2016 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
-  *
-  * Localization strings for the ru_RU (Русский) locale.
-  * With updates from Pavel Pisklakov (revising translation). 
+  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=Translating_OxS
```
Comment (ppv1979) [inline locale/ru_RU/admin.xml]: Ok.Can you do it for all files?

**Comment by ppv1979** [comment]
@mtub, please, clean up unwanted commits by yourself.

File: locale/ru_RU/admin.xml (hunk 3 above)
Comment (mtub) [inline locale/ru_RU/admin.xml]: Yes, I will do it.

**Pull request closed without merging**